site/docs/uk/advanced/middleware.md

```diff
@@ -0,0 +1,121 @@
+---
+prev: ./
+next: ./structuring.md
+---
+
+# Можливості проміжних обробників
+
+У посібнику [ми представили проміжні обробники](../guide/middleware.md) як стек функцій.
+Хоча ви можете використовувати проміжні обробники у такий лінійний спосіб, називати їх просто стеком - це спрощення.
```

Comment:
Should the hyphen (-) be an &mdash; (—)?

Reply:
I think no because we don't use a long dash, but let's go wait for mind @Nazar-Ant, @deptyped.

Reply:
In fact, we use both kinds of hyphens and they are inconsistent in some places. Fixed.